docs/builtin.rst

```diff
@@ -77,6 +77,12 @@ Available
      - ``junifer.configs.juseless``
      - In Progress
      - :gh:`57`
+   * - :class:`junifer.configs.juseless.JuselessDataladIXIVBM`
```

Comment:
I think it should be: `junifer.configs.juseless.datagrabbers.JuselessDataladIXIVBM`

I'll make the necessary changes for the other juseless stuff.